libnavui-dropin/src/main/res/layout/mapbox_info_panel_layout.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:background="@drawable/mapbox_bg_info_panel"
+    android:elevation="8dp">
+
+    <FrameLayout
+        android:id="@+id/infoPanelHeader"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/mapbox_infoPanel_peekHeight"
```

Comment:
I think it should be okay to set it to `wrap_content`. Doing that results in the screenshot attached below.

<img src="https://user-images.githubusercontent.com/9770186/179063338-7149f5df-c44f-4ede-9e18-b1105325e849.jpg" width="200"/>


Maybe I am missing something obvious, but I think this is an issue. Because, in this case we are adding a new layout to `header binder` in the form of `qa_test_app/res/layout_info_panel_header` which has a `constraintLayout` with height of `104dp`. So how come the above `FrameLayout` `height` doesn't expand to `104dp`, but rather gets limited to the height of `textView`?

Reply:
In what OS version did you notice this?
I have tested this in an _emulator with Android 6.0 (API 23)_, a  _device with Android 11 (API 30)_  and a _device with Android 12 (API 32)_. It works as expected.
|emulator with Android 6.0 <br/>(API 23)| Google Pixel 2 Android 11 <br/>(API 30)  | Google Pixel 6 Android 12 <br/>(API 32) |
|:---:|:---:|:---:|
|![emulator-android6-api23](https://user-images.githubusercontent.com/2678039/179244722-39727e9b-9f92-482e-b4ee-4bcb67db9d88.png) | ![pixel2](https://user-images.githubusercontent.com/2678039/179242946-100008db-9534-4ceb-a3ac-60faafe8e260.png) | ![pixel6](https://user-images.githubusercontent.com/2678039/179242950-fc36f8d8-1019-4421-bb07-0c63e667dc91.png) |

Reply:
@tomaszrybakiewicz You tested it with `FrameLayout` height set to `wrap_content` instead of `@dimen/mapbox_infoPanel_peekHeight`. The image I shared above happens when I set the height of the `FrameLayout` to `wrap_content`

Reply:
OK. I figured out what's going on here.

I forgot to pass **parent layout** [when inflating header view in the custom InfoPanelHeaderBinder](https://github.com/mapbox/mapbox-navigation-android/pull/6049/files#diff-bad7504047ae8e24d4e03581ce6693a1eb7d3a4b6831e94abe071f2f89d8c9f5R381).
This resulted in an ambiguous layout where a child with `android:layout_height="match_parent"` is  added to a parent with `android:layout_height="wrap_content"`.

I patched it up. Now it should work as expected.